app/core/config.py

```diff
@@ -36,6 +36,10 @@
 )
 DEFAULT_APP = config("DEFAULT_APP", cast=str, default="entity-viewer")
 ENSEMBL_URL = config("ENSEMBL_URL", cast=str, default="https://beta.ensembl.org")
+NCBI_DATASETS_URL = config(
+    "NCBI_DATASETS_URL", cast=str, default="https://api.ncbi.nlm.nih.gov/datasets/v2"
+)
```

Comment:
Why should this be read from an environment variable? NCBI url has nothing to do with our environments.

Reply:
Seems appropriate to have an environment variable for data like an external URL like this.